testing/libfixposix/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+# Contributor: Will Sinatra <wpsinatra@gmail.com>
+pkgname=libfixposix
+pkgver=0.4.3
+pkgrel=1.0
+pkgdesc="Thin wrapper over POSIX syscalls"
+url="https://github.com/sionescu/libfixposix"
+arch="x86_64"
+license="BSL-1.0"
+makedepends="autoconf automake libtool"
+source="$pkgname-$pkgver.tar.gz::https://github.com/sionescu/libfixposix/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare () {
+	cd "$builddir"
+	autoreconf -f -i
+}
+
+check () {
+      cd "$builddir"
+      make check
+}      
+	
+build () {
+      cd "$builddir"
+      ./configure --prefix=/usr
+      make
+}      
+
+package() {
+	cd "$builddir"
+	make DESTDIR="$pkgdir" install
+	libtool --finish /usr/lib
```

Comment:
this libtool will run against the system or container /usr/lib, can't this be removed ?